pkg/nvml/api.go

```diff
@@ -292,7 +292,7 @@ type Interface interface {
 	GpmMetricsGetV(MetricsGet *GpmMetricsGetType) GpmMetricsGetVType
 	GpmMetricsGet(MetricsGet *GpmMetricsGetType) Return
 	GpmSampleFree(GpmSample GpmSample) Return
-	GpmSampleAlloc(GpmSample *GpmSample) Return
+	GpmSampleAlloc() (GpmSample, Return)
```

Comment:
Does it make sense to move this to a separate PR to call it out properly?

Reply:
Ditto here, I will look into it as a "pre" PR.

Reply:
It's not a blocker -- as long as we actually call it out in the release notes.